starknet_devnet/devnet_config.py

```diff
@@ -66,6 +67,18 @@ def _fork_block(specifier: str):
     return parsed
 
 
+def _chain_id(chain_id: str):
+    """Parse chain id.'"""
+    chain_ids = [member.name for member in StarknetChainId]
+
+    try:
+        chain_id = StarknetChainId[chain_id].name
+    except KeyError:
+        sys.exit(f"The value of --chain_id must be in {chain_ids}, got: {chain_id}")
```

Comment:
I would prepend `Error: ` here to follow the same pattern as with other validation places. I would advise taking this as an example: https://github.com/Shard-Labs/starknet-devnet/blob/7c11183ae76073363aca9bac69483e9aaf55802e/starknet_devnet/devnet_config.py#L142

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/358/commits/9f4d0ede45cfb89c29ab8c653895d01c84349002